tox.ini

```diff
@@ -67,10 +67,8 @@ whitelist_externals =
 deps =
     -r{toxinidir}/requirements/quality.txt
 commands =
-    touch tests/__init__.py
```

Comment:
Remove the touch/rm pair, and removed `tests` and `test_utils` from each of these lines.

Since we don't use those directories in a number of our repos (we use internal tests dirs), maybe we should remove them from the cookiecutter templates?